[🐛 Bug]:  Helm post-upgrade hook fails when recreating selenium-grid-edge-scaledobject

### What happened?

I'm deploying selenium-grid helm chart on AKS cluster with an existing keda (helm chart version 2.12.1) installed using Azure pipelines. The helm upgrade job fails with the following error:

Error: UPGRADE FAILED: post-upgrade hooks failed: warning: Hook post-upgrade selenium-grid/templates/edge-node-hpa.yaml failed: object is being deleted: scaledobjects.keda.sh "selenium-grid-edge-scaledobject" already exists 

Although after rerunning the failed job, the helm upgrade is successful. The selenium-grid-chrome-scaledobject gets deployed without any issues at the first time, with the selenium-grid-edge-scaledobject there's always this rerun needed. Is there any way how to control the post-upgrade hook deployments, to set some kind of delay on this particular hook? Maybe setting the hook-weight and changing the order of the chrome (by default is installed first) and edge scaled object deployment could solve the issue. With the current selenium-grid helm chart I'm able to set only identical annotations for both scaled objects.

### Command used to start Selenium Grid with Docker (or Kubernetes)

```shell
helm upgrade --install --values deployments/selenium-grid/envs/DEV/deployment.yaml --wait --timeout 5m0s --create-namespace -n platform-test selenium-grid repo_selenium-grid/selenium-grid --version 0.25.3

deployments/selenium-grid/envs/DEV/deployment.yaml

basicAuth:
  enabled: false
ingress:
  enabled: false
autoscaling:
  enableWithExistingKEDA: true
  scalingType: deployment
  scaledOptions:
    minReplicaCount: 0
    maxReplicaCount: 8
    advanced:
      horizontalPodAutoscalerConfig:
        behavior:
          scaleDown:
            stabilizationWindowSeconds: 300
            policies:
              - type: Percent
                value: 100
                periodSeconds: 15

hub:
  serviceType: ClusterIP
  nodeSelector: { agentpool: "seleniumpool" }
  tolerations:
    - effect: NoSchedule
      key: role
      operator: Equal
      value: seleniumNode
  extraVolumeMounts:
    - name: qa-file-pv
      mountPath: /mnt/testautomation-files
    - name: selenium-kv-secrets-store-inline
      mountPath: "/mnt/secrets-store"
      readOnly: true
  extraVolumes:
    - name: qa-file-pv
      persistentVolumeClaim:
        claimName: qa-file-pvc
    - name: selenium-kv-secrets-store-inline
      csi:
        driver: secrets-store.csi.k8s.io
        readOnly: true
        volumeAttributes:
          secretProviderClass: selenium-kv-secrets-provider
  extraEnvFrom:
    - secretRef:
        name: selenium-kv-secret

# Configuration for chrome nodes
chromeNode:
  nodeSelector: { agentpool: "seleniumpool" }
  tolerations:
    - effect: NoSchedule
      key: role
      operator: Equal
      value: seleniumNode
  extraVolumeMounts:
    - name: qa-file-pv
      mountPath: /mnt/testautomation-files
    - name: selenium-kv-secrets-store-inline
      mountPath: "/mnt/secrets-store"
      readOnly: true
  extraVolumes:
    - name: qa-file-pv
      persistentVolumeClaim:
        claimName: qa-file-pvc
    - name: selenium-kv-secrets-store-inline
      csi:
        driver: secrets-store.csi.k8s.io
        readOnly: true
        volumeAttributes:
          secretProviderClass: selenium-kv-secrets-provider
  extraEnvFrom:
    - secretRef:
        name: selenium-kv-secret
  scaledObjectOptions:
    scaleTargetRef:
      name: selenium-chrome-node

# Configuration for firefox nodes
firefoxNode:
  # Enable firefox nodes
  enabled: false

# Configuration for edge nodes
edgeNode:
  nodeSelector: { agentpool: "seleniumpool" }
  tolerations:
    - effect: NoSchedule
      key: role
      operator: Equal
      value: seleniumNode
  extraVolumeMounts:
    - name: qa-file-pv
      mountPath: /mnt/testautomation-files
    - name: selenium-kv-secrets-store-inline
      mountPath: "/mnt/secrets-store"
      readOnly: true
  extraVolumes:
    - name: qa-file-pv
      persistentVolumeClaim:
        claimName: qa-file-pvc
    - name: selenium-kv-secrets-store-inline
      csi:
        driver: secrets-store.csi.k8s.io
        readOnly: true
        volumeAttributes:
          secretProviderClass: selenium-kv-secrets-provider
  extraEnvFrom:
    - secretRef:
        name: selenium-kv-secret
  scaledObjectOptions:
    scaleTargetRef:
      name: selenium-edge-node
```


### Relevant log output

```shell
Error: UPGRADE FAILED: post-upgrade hooks failed: warning: Hook post-upgrade selenium-grid/templates/edge-node-hpa.yaml failed: object is being deleted: scaledobjects.keda.sh "selenium-grid-edge-scaledobject" already exists
```


### Operating System

AKS 1.27.9

### Docker Selenium version (image tag)

4.15.0-20231129

### Selenium Grid chart version (chart version)

0.25.3

I just realized I'm not able to reproduce it by manually deploying via cli, I get this error in the pipeline exclusively, here's the raw log with timestamps from the last run, not sure if this is any helpful:

```
2024-04-15T12:40:02.7192162Z [command]/azp/_work/_tool/helm/3.9.0/x64/linux-amd64/helm upgrade --install --values /azp/_work/1/s/helm/deployments/selenium-grid/configurations/envs/DEV/deployment.yaml --wait --timeout 5m0s --create-namespace -n platform-test selenium-grid repo_selenium-grid/selenium-grid --version 0.29.0
2024-04-15T12:40:08.9650508Z Error: UPGRADE FAILED: post-upgrade hooks failed: warning: Hook post-upgrade selenium-grid/templates/edge-node-hpa.yaml failed: scaledobjects.keda.sh "selenium-edge-node" already exists
2024-04-15T12:40:08.9807233Z ##[error]Error: UPGRADE FAILED: post-upgrade hooks failed: warning: Hook post-upgrade selenium-grid/templates/edge-node-hpa.yaml failed: scaledobjects.keda.sh "selenium-edge-node" already exists

2024-04-15T12:40:08.9820775Z ##[section]Finishing: helm upgrade
```

Sorry, this is the last one, it appears it's slightly different:

```
2024-04-18T11:22:25.5768644Z [command]/azp/_work/_tool/helm/3.9.0/x64/linux-amd64/helm upgrade --install --values /azp/_work/1/s/helm/deployments/selenium-grid/configurations/envs/DEV/deployment.yaml --wait --timeout 5m0s --create-namespace -n platform-test selenium-grid repo_selenium-grid/selenium-grid --version 0.29.0
2024-04-18T11:22:30.9688654Z Error: UPGRADE FAILED: post-upgrade hooks failed: warning: Hook post-upgrade selenium-grid/templates/edge-node-hpa.yaml failed: object is being deleted: scaledobjects.keda.sh "selenium-edge-node" already exists
2024-04-18T11:22:30.9730983Z ##[error]Error: UPGRADE FAILED: post-upgrade hooks failed: warning: Hook post-upgrade selenium-grid/templates/edge-node-hpa.yaml failed: object is being deleted: scaledobjects.keda.sh "selenium-edge-node" already exists

2024-04-18T11:22:30.9861278Z ##[section]Finishing: helm upgrade
```

--- a/.github/workflows/build-test.yml
+++ b/.github/workflows/build-test.yml
@@ -51,7 +51,11 @@ jobs:
         env:
           AUTHORS: ${{ vars.AUTHORS || 'SeleniumHQ' }}
       - name: Build Docker images
-        run: VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make build
+        uses: nick-invision/retry@master
+        with:
+          timeout_minutes: 12
+          max_attempts: 3
+          command: VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make build
       - name: Count image layers
         run: VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make count_image_layers
       - name: Test Docker images

--- a/.github/workflows/helm-chart-test.yml
+++ b/.github/workflows/helm-chart-test.yml
@@ -116,7 +116,11 @@ jobs:
           echo "CHART_PACKAGE_PATH=$(cat /tmp/selenium_chart_version)" >> $GITHUB_ENV
           echo "CHART_FILE_NAME=$(basename $(cat /tmp/selenium_chart_version))" >> $GITHUB_ENV
       - name: Build Docker images
-        run: NAME=${IMAGE_REGISTRY} VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make build
+        uses: nick-invision/retry@master
+        with:
+          timeout_minutes: 12
+          max_attempts: 3
+          command: NAME=${IMAGE_REGISTRY} VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make build
       - name: Setup Kubernetes cluster
         uses: nick-invision/retry@master
         with:

--- a/.github/workflows/test-video.yml
+++ b/.github/workflows/test-video.yml
@@ -67,11 +67,15 @@ jobs:
         env:
           AUTHORS: ${{ vars.AUTHORS || 'SeleniumHQ' }}
       - name: Pre-build to reduce logs in test phase
-        run: |
-          VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make hub
-          VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make chrome
-          VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make firefox
-          VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make edge
+        uses: nick-invision/retry@master
+        with:
+          timeout_minutes: 12
+          max_attempts: 3
+          command: |
+            VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make hub
+            VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make chrome
+            VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make firefox
+            VERSION=${BRANCH} BUILD_DATE=${BUILD_DATE} make edge
       - name: Set test parameters
         if: (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch')
         run: |

--- a/Base/Dockerfile
+++ b/Base/Dockerfile
@@ -55,7 +55,7 @@ RUN  echo "deb http://archive.ubuntu.com/ubuntu jammy main universe\n" > /etc/ap
     gnupg2 \
     libnss3-tools \
   && mkdir -p /etc/apt/keyrings \
-  && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 843C48A565F8F04B \
+  && apt-key adv --keyserver hkps://keyserver.ubuntu.com:443 --recv-keys 843C48A565F8F04B || apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 843C48A565F8F04B \
   && wget -qO - https://packages.adoptium.net/artifactory/api/gpg/key/public | tee /etc/apt/keyrings/adoptium.asc >dev/null \
   && echo "deb [signed-by=/etc/apt/keyrings/adoptium.asc] https://packages.adoptium.net/artifactory/deb $(awk -F= '/^VERSION_CODENAME/{print$2}' /etc/os-release) main" | tee /etc/apt/sources.list.d/adoptium.list >dev/null \
   && apt-get -qqy update \

--- a/charts/selenium-grid/templates/_nameHelpers.tpl
+++ b/charts/selenium-grid/templates/_nameHelpers.tpl
@@ -39,6 +39,13 @@ app.kubernetes.io/component: {{ printf "selenium-grid-%s" .Chart.AppVersion }}
 helm.sh/chart: {{ include "seleniumGrid.chart" . }}
 {{- end -}}
 
+{{/*
+Autoscaling labels
+*/}}
+{{- define "seleniumGrid.autoscalingLabels" -}}
+component.autoscaling: "true"
+{{- end -}}
+
 {{- define "seleniumGrid.component.name" -}}
 {{- $component := index . 0 }}
 {{- $root := index . 1 }}
@@ -180,4 +187,25 @@ Server ConfigMap fullname
 */}}
 {{- define "seleniumGrid.server.configmap.fullname" -}}
 {{- tpl (default (include "seleniumGrid.component.name" (list "selenium-server-config" $)) .Values.serverConfigMap.nameOverride) $ | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Patch scaledObjects finalizers job fullname
+*/}}
+{{- define "seleniumGrid.keda.patchObjectsJob.fullname" -}}
+{{- printf "%s-%s" .Release.Name "patch-scaledobjects-finalizers" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Patch scaled objects RoleBinding fullname
+*/}}
+{{- define "seleniumGrid.keda.roleBinding.fullname" -}}
+{{- printf "%s-%s" .Release.Name "patch-keda-rb" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Patch scaled objects Role fullname
+*/}}
+{{- define "seleniumGrid.keda.role.fullname" -}}
+{{- printf "%s-%s" .Release.Name "patch-keda-role" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}

--- a/charts/selenium-grid/templates/chrome-node-hpa.yaml
+++ b/charts/selenium-grid/templates/chrome-node-hpa.yaml
@@ -11,6 +11,7 @@ metadata:
   labels:
     deploymentName: {{ template "seleniumGrid.chromeNode.fullname" . }}
     {{- include "seleniumGrid.commonLabels" . | nindent 4 }}
+    {{- include "seleniumGrid.autoscalingLabels" . | nindent 4 }}
     {{- with .Values.chromeNode.labels }}
       {{- toYaml . | nindent 4 }}
     {{- end }}

--- a/charts/selenium-grid/templates/chrome-node-scaledjobs.yaml
+++ b/charts/selenium-grid/templates/chrome-node-scaledjobs.yaml
@@ -12,6 +12,7 @@ metadata:
     app: {{ template "seleniumGrid.chromeNode.fullname" . }}
     app.kubernetes.io/name: {{ template "seleniumGrid.chromeNode.fullname" . }}
     {{- include "seleniumGrid.commonLabels" . | nindent 4 }}
+    {{- include "seleniumGrid.autoscalingLabels" . | nindent 4 }}
     {{- with .Values.chromeNode.labels }}
       {{- toYaml . | nindent 4 }}
     {{- end }}

--- a/charts/selenium-grid/templates/edge-node-hpa.yaml
+++ b/charts/selenium-grid/templates/edge-node-hpa.yaml
@@ -11,6 +11,7 @@ metadata:
   labels:
     deploymentName: {{ template "seleniumGrid.edgeNode.fullname" . }}
     {{- include "seleniumGrid.commonLabels" . | nindent 4 }}
+    {{- include "seleniumGrid.autoscalingLabels" . | nindent 4 }}
     {{- with .Values.edgeNode.labels }}
       {{- toYaml . | nindent 4 }}
     {{- end }}

--- a/charts/selenium-grid/templates/edge-node-scaledjob.yaml
+++ b/charts/selenium-grid/templates/edge-node-scaledjob.yaml
@@ -12,6 +12,7 @@ metadata:
     app: {{ template "seleniumGrid.edgeNode.fullname" . }}
     app.kubernetes.io/name: {{ template "seleniumGrid.edgeNode.fullname" . }}
     {{- include "seleniumGrid.commonLabels" . | nindent 4 }}
+    {{- include "seleniumGrid.autoscalingLabels" . | nindent 4 }}
     {{- with .Values.edgeNode.labels }}
       {{- toYaml . | nindent 4 }}
     {{- end }}

--- a/charts/selenium-grid/templates/firefox-node-hpa.yaml
+++ b/charts/selenium-grid/templates/firefox-node-hpa.yaml
@@ -11,6 +11,7 @@ metadata:
   labels:
     deploymentName: {{ template "seleniumGrid.firefoxNode.fullname" . }}
     {{- include "seleniumGrid.commonLabels" . | nindent 4 }}
+    {{- include "seleniumGrid.autoscalingLabels" . | nindent 4 }}
     {{- with .Values.firefoxNode.labels }}
       {{- toYaml . | nindent 4 }}
     {{- end }}

--- a/charts/selenium-grid/templates/firefox-node-scaledjob.yaml
+++ b/charts/selenium-grid/templates/firefox-node-scaledjob.yaml
@@ -12,6 +12,7 @@ metadata:
     app: {{ template "seleniumGrid.firefoxNode.fullname" . }}
     app.kubernetes.io/name: {{ template "seleniumGrid.firefoxNode.fullname" . }}
     {{- include "seleniumGrid.commonLabels" . | nindent 4 }}
+    {{- include "seleniumGrid.autoscalingLabels" . | nindent 4 }}
     {{- with .Values.firefoxNode.labels }}
       {{- toYaml . | nindent 4 }}
     {{- end }}

--- a/charts/selenium-grid/templates/patch-keda/patch-keda-objects-job.yaml
+++ b/charts/selenium-grid/templates/patch-keda/patch-keda-objects-job.yaml
@@ -1,0 +1,35 @@
+{{- if eq (include "seleniumGrid.useKEDA" $) "true" }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "seleniumGrid.keda.patchObjectsJob.fullname" $ }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook-delete-policy": hook-succeeded
+    "helm.sh/resource-policy": delete
+    {{- with $.Values.autoscaling.annotations }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+  labels:
+    deploymentName: {{ template "seleniumGrid.keda.patchObjectsJob.fullname" $ }}
+    {{- include "seleniumGrid.commonLabels" $ | nindent 4 }}
+    {{- with $.Values.customLabels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  template:
+    metadata:
+      name: {{ template "seleniumGrid.keda.patchObjectsJob.fullname" $ }}
+    spec:
+      activeDeadlineSeconds: 120
+      serviceAccountName: {{ template "seleniumGrid.serviceAccount.fullname" $ }}
+      serviceAccount: {{ template "seleniumGrid.serviceAccount.fullname" $ }}
+      containers:
+        - name: {{ template "seleniumGrid.keda.patchObjectsJob.fullname" $ }}
+          image: {{ $.Values.global.seleniumGrid.kubectlImage }}
+          command:
+            - "bin/bash"
+            - "-c"
+            - "kubectl get ScaledObjects,ScaledJobs -n {{ .Release.Namespace }} -l component.autoscaling=true -o=json | jq '.metadata.finalizers = null' | kubectl apply -f -"
+      restartPolicy: Never
+{{- end }}

--- a/charts/selenium-grid/templates/patch-keda/patch-keda-rb.yaml
+++ b/charts/selenium-grid/templates/patch-keda/patch-keda-rb.yaml
@@ -1,0 +1,20 @@
+{{- if eq (include "seleniumGrid.useKEDA" $) "true" }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "seleniumGrid.keda.roleBinding.fullname" $ }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    deploymentName: {{ template "seleniumGrid.keda.roleBinding.fullname" $ }}
+    {{- include "seleniumGrid.commonLabels" $ | nindent 4 }}
+    {{- with $.Values.customLabels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "seleniumGrid.serviceAccount.fullname" $ }}
+roleRef:
+  kind: Role
+  name: {{ template "seleniumGrid.keda.role.fullname" $ }}
+  apiGroup: rbac.authorization.k8s.io
+{{- end }}

--- a/charts/selenium-grid/templates/patch-keda/patch-keda-role.yaml
+++ b/charts/selenium-grid/templates/patch-keda/patch-keda-role.yaml
@@ -1,0 +1,37 @@
+{{- if eq (include "seleniumGrid.useKEDA" $) "true" }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ template "seleniumGrid.keda.role.fullname" $ }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    deploymentName: {{ template "seleniumGrid.keda.role.fullname" $ }}
+    {{- include "seleniumGrid.commonLabels" $ | nindent 4 }}
+    {{- with $.Values.customLabels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+rules:
+  - apiGroups:
+      - keda.sh
+    resources:
+      - scaledjobs
+      - scaledjobs/finalizers
+      - scaledjobs/status
+    verbs:
+      - '*'
+  - apiGroups:
+      - keda.sh
+    resources:
+      - scaledobjects
+      - scaledobjects/finalizers
+      - scaledobjects/status
+    verbs:
+      - '*'
+  - apiGroups:
+      - keda.sh
+    resources:
+      - triggerauthentications
+      - triggerauthentications/status
+    verbs:
+      - '*'
+{{- end }}

--- a/charts/selenium-grid/values.yaml
+++ b/charts/selenium-grid/values.yaml
@@ -12,6 +12,8 @@ global:
     nodesImageTag: 4.19.1-20240402
     # Image tag for browser's video recorder
     videoImageTag: ffmpeg-7.0-20240402
+    # kubectl image is used to execute kubectl commands in utility jobs
+    kubectlImage: bitnami/kubectl:latest
     # Pull secret for all components, can be overridden individually
     imagePullSecret: ""
     # Log level for all components. Possible values describe here: https://www.selenium.dev/documentation/grid/configuration/cli_options/#logging
@@ -546,7 +548,7 @@ autoscaling:
   scalingType: job
   # Annotations for KEDA resources: ScaledObject and ScaledJob
   annotations:
-    "helm.sh/hook": post-install
+    "helm.sh/hook": post-install,post-upgrade,post-rollback,pre-delete
   # Options for KEDA scaled resources (keep only common options used for both ScaledJob and ScaledObject)
   scaledOptions:
     minReplicaCount: 0
@@ -1185,7 +1187,7 @@ customLabels: {}
 # Configuration for dependency chart keda
 keda:
   additionalAnnotations:
-    "helm.sh/hook": pre-install
+    "helm.sh/hook": pre-install,pre-upgrade,pre-rollback,post-delete
   http:
     timeout: 60000
   webhooks:
@@ -1221,19 +1223,19 @@ prometheus-stack:
   defaultRules:
     create: true
     annotations:
-      "helm.sh/hook": post-install
+      "helm.sh/hook": post-install,post-upgrade,post-rollback,pre-delete
   alertmanager:
     enabled: true
     annotations:
-      "helm.sh/hook": post-install
+      "helm.sh/hook": post-install,post-upgrade,post-rollback,pre-delete
   grafana:
     enabled: true
     adminPassword: admin
     forceDeployDatasources: true
     forceDeployDashboards: true
     annotations:
-      "helm.sh/hook": post-install
+      "helm.sh/hook": post-install,post-upgrade,post-rollback,pre-delete
   prometheus:
     enabled: true
     annotations:
-      "helm.sh/hook": post-install
+      "helm.sh/hook": post-install,post-upgrade,post-rollback,pre-delete

--- a/tests/charts/make/chart_test.sh
+++ b/tests/charts/make/chart_test.sh
@@ -97,19 +97,19 @@ HELM_COMMAND_SET_IMAGES=" \
 --set global.seleniumGrid.logLevel=${LOG_LEVEL} \
 "
 
-if [ "${TEST_EXISTING_KEDA}" = "true" ]; then
+if [ "${SELENIUM_GRID_AUTOSCALING}" = "true" ] && [ "${TEST_EXISTING_KEDA}" = "true" ]; then
   HELM_COMMAND_SET_IMAGES="${HELM_COMMAND_SET_IMAGES} \
   --set autoscaling.enabled=false \
   --set autoscaling.enableWithExistingKEDA=true \
   "
-else
+elif [ "${SELENIUM_GRID_AUTOSCALING}" = "true" ] && [ "${TEST_EXISTING_KEDA}" = "true" ]; then
   HELM_COMMAND_SET_IMAGES="${HELM_COMMAND_SET_IMAGES} \
   --set autoscaling.enabled=true \
   --set autoscaling.enableWithExistingKEDA=false \
   "
 fi
 
-if [ -n "${SET_MAX_REPLICAS}" ]; then
+if [ "${SELENIUM_GRID_AUTOSCALING}" = "true" ] && [ -n "${SET_MAX_REPLICAS}" ]; then
   HELM_COMMAND_SET_IMAGES="${HELM_COMMAND_SET_IMAGES} \
   --set autoscaling.scaledOptions.maxReplicaCount=${SET_MAX_REPLICAS} \
   "
@@ -151,7 +151,6 @@ fi
 
 if [ "${SELENIUM_GRID_AUTOSCALING}" = "true" ]; then
   HELM_COMMAND_SET_AUTOSCALING=" \
-  --set autoscaling.enableWithExistingKEDA=${SELENIUM_GRID_AUTOSCALING} \
   --set autoscaling.scaledOptions.minReplicaCount=${SELENIUM_GRID_AUTOSCALING_MIN_REPLICA} \
   "
 fi

